src/applications/personalization/appointments/actions/index.js

```diff
@@ -1,27 +1,216 @@
+import moment from 'moment';
+import { replace, uniq, map, isEmpty, reduce } from 'lodash';
```

Comment:
If you're using `map` and `reduce` with arrays, I'd really prefer using the built in Array methods. I'm not opposed to using lodash, I just prefer using native methods if they are available and do the job.

Reply:
Okay! I changed it to the native `map` and `reduce`. I always feel safer using lodash knowing my code won't throw if any values are `undefined`.